hdKeysGenerator/main_test.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"context"
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-tools-go/hdKeysGenerator/common"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	// See https://raw.githubusercontent.com/ElrondNetwork/elrond-sdk-testwallets/main/users/mnemonic.txt
+	DummyMnemonic = "moral volcano peasant pass circle pen over picture flat shop clap goat never lyrics gather prepare woman film husband gravity behind test tiger improve"
+)
+
+func TestInlineSortKeysByIndexes(t *testing.T) {
```

Comment:
moving the login of generateKeys & generateKeysInParallel would also benefit from the fact that you would not be forced to have main_test